lexicons/bsky.app/updateProfile.json

```diff
@@ -0,0 +1,37 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.updateProfile",
+  "type": "procedure",
+  "description": "Notify server that the user has seen notifications",
+  "parameters": { },
+  "input": {
+    "encoding": "application/json",
+    "schema": {
+      "type": "object",
+      "required": [],
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "maxLength": 64
+        },
+        "description": {
+          "type": "string",
+          "maxLength": 256
+        },
+        "badges": {"type": "array", "items": {"$ref": "lex:app.bsky.profile#/defs/badgeRef"}}
```

Comment:
Do we need to indicate this is, like, highlightedBadges?

Reply:
Or pinnedBadges?

Reply:
eh yeah probably a good idea